newIDE/app/src/SceneEditor/index.js

```diff
@@ -598,24 +598,19 @@ export default class SceneEditor extends React.Component<Props, State> {
     const { project, layout } = this.props;
 
     //eslint-disable-next-line
-    const answer = confirm(
-      'Do you want to remove all references to this object in groups and events (actions and conditions using the object)?'
-    );
 
     if (global) {
       gd.WholeProjectRefactorer.globalObjectOrGroupRemoved(
         project,
         object.getName(),
-        /* isObjectGroup=*/ false,
-        !!answer
```

Comment:
Not passing an argument to gd.xxx methods is very dangerous, this can crash the C++ library.

Reply:
Oh. I should've investigated about these methods before blindly removing the arguments. Pardon me for this carelessness.